From 4e6a9873e281b133e7fa1c81be9059b0ea792271 Mon Sep 17 00:00:00 2001
From: Kishan Mochi <kishan.mochi@intel.com>
Date: Mon, 19 Oct 2020 14:16:51 +0530
Subject: [PATCH] AOSP R41 Rebase code changes for packages/apps

This patch is for AOSP R41 rebase code changes and
to fix the compilation issue

Change-Id: I08eed18a35c01e18540b5798d89793464b506d36
Tracked-On: OAM-94364
Signed-off-by: Kishan Mochi <kishan.mochi@intel.com>

diff --git a/src/com/android/settings/datausage/DataSaverBackend.java b/src/com/android/settings/datausage/DataSaverBackend.java
index 92b2ef9a70..a0108e29b4 100644
--- a/src/com/android/settings/datausage/DataSaverBackend.java
+++ b/src/com/android/settings/datausage/DataSaverBackend.java
@@ -203,7 +203,7 @@ public class DataSaverBackend {
         public void onSubscriptionOverride(int subId, int overrideMask, int overrideValue) {
         }
 
-        @Override
+	@Override
         public void onSubscriptionPlansChanged(int subId, SubscriptionPlan[] plans) {
         }
     };
diff --git a/src/com/android/settings/password/PasswordUtils.java b/src/com/android/settings/password/PasswordUtils.java
index 5b75033f0f..1ead492788 100644
--- a/src/com/android/settings/password/PasswordUtils.java
+++ b/src/com/android/settings/password/PasswordUtils.java
@@ -89,8 +89,7 @@ public final class PasswordUtils extends com.android.settingslib.Utils {
                     /* initialPid= */ -1,
                     getCallingAppPackageName(activityToken),
                     userId,
-                    message,
-                    false);
+                    message);
         } catch (RemoteException e) {
             Log.v(TAG, "Could not talk to activity manager.", e);
         }
-- 
2.28.0

